pkg/api/types.go

```diff
@@ -715,6 +719,113 @@ func (p *Properties) HasVirtualMachineScaleSets() bool {
 	return false
 }
 
+// K8sOrchestratorName returns the 3 character orchestrator code for kubernetes-based clusters.
+func (p *Properties) K8sOrchestratorName() string {
+	if p.OrchestratorProfile.IsKubernetes() ||
+		p.OrchestratorProfile.IsOpenShift() {
+		if p.HostedMasterProfile != nil {
+			return "aks"
```

Comment:
this should be a const as well

is it a new value introduced?

Reply:
also should `(p.OrchestratorProfile.IsOpenShift() && p.HostedMasterProfile != nil)` return `aks`?

Reply:
also should `(p.OrchestratorProfile.IsOpenShift() && p.HostedMasterProfile != nil)` return `aks`?

Reply:
no. You'll find this is params_k8s.go

Reply:
> also should (p.OrchestratorProfile.IsOpenShift() && p.HostedMasterProfile != nil) return aks?
This is the current behavior so we should leave it that way